odl/solvers/advanced/proximal_operators.py

```diff
@@ -706,6 +755,30 @@ def proximal_cconj_l2_squared(space, lam=1, g=None):
     prox_factory : function
         Factory for the proximal operator to be initialized
 
+    Notes
+    -----
+    The functional :math:`F` is given by
+
+        .. math::
+
+            F(x) =  \\lambda ||x - g||_2^2.
+
+    The convex conjugate :math:`F^*` of :math:`F` is given by
+
+        .. math::
+
+            F^*(y) = \\frac{1}{\\lambda} \left( \left|\left|
+            \\frac{y}{\\lambda} \\right| \\right|_2^2 +
```

Comment:
But the expression for the proximal was still correct (which is what I hoped/expected, since it is the same as implemented and it passes the tests)